usaspending_api/config/tests/unit/test_config.py

```diff
@@ -493,6 +337,389 @@ def factory_func():
     },
 ]
 
+def test_config_values():
+    pprint(CONFIG.dict())
+    print(CONFIG.POSTGRES_DSN)
+    print(str(CONFIG.POSTGRES_DSN))
+
+
+def test_config_loading():
+    with mock.patch.dict(os.environ, {ENV_CODE_VAR: LocalConfig.ENV_CODE}):
+        _load_config.cache_clear()  # wipes the @lru_cache for fresh run on next call
+        cfg = _load_config()
+        pprint(cfg.dict())
+
+
+def test_cannot_instantiate_default_settings():
+    with pytest.raises(NotImplementedError):
+        DefaultConfig()
+
+
+def test_can_instantiate_non_default_settings():
+    LocalConfig()
```

Comment:
Should this be testing that the CONFIG values are expected values associated with LocalConfig()?